src/SubtitlesParser/index.js

```diff
@@ -0,0 +1,216 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 Metrological
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default class SubtitlesParser {
+  // @ params url: subtitle file URL
+  // @return parsed subtitles as list of objects
+  // also stores parsed data
+  constructor() {
+    SubtitlesParser.removeSubtitleTextStyles = false
+    SubtitlesParser.clearCurrentSubtitle()
```

Comment:
Why not `this.removeSubtitleTextStyles` and `this.clearCurrentSubtitle()` ?

Reply:
`removeSubtitleTextStyles` and `clearCurrentSubtitle` are static methods/ properties which cannot be accessed using this.